azext_iot/central/params.py

```diff
@@ -57,13 +69,92 @@ def load_central_arguments(self, _):
         )
         context.argument(
             "central_dns_suffix",
-            options_list=["--central-dns-suffix"],
+            options_list=["--central-dns-suffix", "--central-api-uri"],
             help="Central dns suffix. "
             "This enables running cli commands against non public/prod environments",
         )
         context.argument(
             "device_status",
-            options_list=["--devicestatus", "--ds"],
+            options_list=["--device-status", "--ds"],
             arg_type=get_enum_type(DeviceStatus),
             help="Indicates filter option for device status",
         )
+
+    with self.argument_context("iot central app monitor-events") as context:
+        context.argument("timeout", arg_type=event_timeout_type)
+        context.argument("properties", arg_type=event_msg_prop_type)
+
+    with self.argument_context("iot central app validate-messages") as context:
+        context.argument("timeout", arg_type=event_timeout_type)
+        context.argument("properties", arg_type=event_msg_prop_type)
+        context.argument(
+            "time_range",
```

Comment:
Why not just name this `duration` or similar?  `--time-range` sounds a bit off to me. Like I'm looking at time series data. `--duration` also seems more inline with the use of `--duration` in other commands.

It would also be good to include the unit in the description.